### PR TITLE
[MIRROR] HoS, warden and officers no longer spawn with a stunbaton in their bags. [MDB IGNORE]

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/tgstation/tgstation/pull/62861
--------------------
## About The Pull Request
The HoS, warden and officers no longer spawn with a stunbaton in their bags.

## Why It's Good For The Game
More often than not, we end up with every security role (minus det) having two stunbatons since both their outfits and their closets have a stunbaton. This entails situations in which power cell drainage or losing the first stunbaton in a way that doesn't result in them becoming the receiving end of the stick would hardly impact their gameplay because of the backup one in their bag. It's also worth mentioning stunbatons are pretty powerful (the nerf they received 2 years ago helped, but the time window between the first and second hit is damn short) and most of their supposed counters (like stims, maint pump-up) barely help against the stamina loss and the knockdown right now.
In short, getting in a melee scuffle an officer can be tough, disposing his baton only for him to whip out another is just frustrating.
This also means they won't be as battle-ready but the chances are they'll take a walk to the brig and gear up anyway. It also adds a scarcity factor. Any extra security baton taken is a baton deprived from new officers, so that security and his officers have a reason to order stunbatons from cargo and interact with one another, but i'm just speculating here.
Here's Orange's blessing:
citrus hath spoken

## Changelog

:cl:
balance: The HoS, warden and officers no longer spawn with a stunbaton in their bags.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
@GoldenAlpharex this is going to cause security to have no batons at all since peacekeeper gear doesn't spawn with one in it

**Comment by Iamgoofball** [comment]
@YakumoChen I'm aware and working on it, I had 'em put a halt on this PR in advance.

**Comment by GoldenAlpharex** [comment]
> @GoldenAlpharex this is going to cause security to have no batons at all since peacekeeper gear doesn't spawn with one in it

~*Good*~

Don't worry, I already got people on the case.
